README.rst

```diff
@@ -95,16 +63,23 @@ Theme
 Compatibility
 =============
 
-Addons created with ``bobtemplates.plone`` are tested to work in Plone 4.3.x and Plone 5.
+Add-ons created with ``bobtemplates.plone`` are tested to work in Plone 4.3.x and Plone 5.
 They should also work with older versions but that was not tested.
 It should work on Linux, Mac and Windows.
 
 
+Documentation
+-------------
+
+Full documentation for end users can be found in the "docs" folder.
+
+It is also available online at http://docs.plone.org/develop/addons/bobtemplates.plone
+
 Installation
-============
+-------------
```

Comment:
This underline for `Installation` is one dash too long, though I guess this does not lead to errors.